### PR TITLE
Line Add on bower.json

### PR DESCRIPTION
**Pull request by marc-casoliva**
This pull requests is useless

### CODE REVIEW

**Comment by xavicorral** [comment]
no ho tinc clar ....

File: bower.json [hunk 1]
```diff
@@ -19,4 +19,5 @@
     "test",
     "tests"
   ]
+
```
Comment (xavicorral) [inline bower.json]: esta linea esta vacia nen

Comment (xavicorral) [inline bower.json]: treu la linea

Comment (marc-casoliva) [inline bower.json]: Always must have a blank line at the end of the files!!! @xavicorral

**Pull request closed without merging**